packages/kbn-es-archiver/src/actions/empty_kibana_index.ts

```diff
@@ -34,6 +34,7 @@ export async function emptyKibanaIndexAction({
   const stats = createStats('emptyKibanaIndex', log);
 
   await deleteKibanaIndices({ client, stats, log });
+  await createDefaultSpace({ client, index: '.kibana' });
```

Comment:
The default space is created when the `.kibana` index is loaded at https://github.com/elastic/kibana/blob/83d9ef0f2d00175915f2c878d45fabb2e4289d90/packages/kbn-es-archiver/src/actions/load.ts#L114, and only when the `spaces` plugin is in use (not in OSS tests). What is this line supposed to be fixing?

Reply:
@spalger I was looking for a way to clean up a Kibana instance between runs and initially I've tried to use `empty_kibana_index`, but after your comment, I found out that there is also `cleanKibanaIndexes` method, which I've exposed in the CLI and seems to be working fine for our case. What do you think about that solution?